test/unit/lib/mapLink.js

```diff
@@ -85,6 +85,57 @@ describe('mapLink', () => {
       expect(results[0].mapUrl).to.be.equal(expectedMapLink);
     });
 
+    it('should leave start address empty for place searches', () => {
+      const yourLocationSearchCriteria = {
+        coordinates,
+        searchTerm,
+        searchType: constants.placeSearch,
+      };
+      const inputList = [{
+        name: 'name',
+        address: {
+          line1: 'line1',
+        },
+      }];
+      const destination = 'name,line1';
+      const params = {
+        saddr: '',
+        daddr: `${destination}`,
+        near: `${destination}`,
+      };
+      const expectedMapLink = `https://maps.google.com/maps?${qs.stringify(params)}`;
+
+      const results = mapLink.addUrl(yourLocationSearchCriteria, inputList);
+
+      expect(results.length).to.be.equal(1);
+      expect(results[0].mapUrl).to.be.equal(expectedMapLink);
+    });
+
+    it('should populate start address for place searches', () => {
```

Comment:
The search in the test is a postcode rather than a place (as stated in the test name).